modules/sql-engine/src/integrationTest/sql/cast/test_cast_bigint.test

```diff
@@ -6,6 +6,10 @@
 statement ok
 SELECT CAST('100' AS BIGINT);
 
+skipif ignite3
```

Comment:
why do we ignore this only for BIGINT type?

Reply:
Filed an issue.